client/modules/datafiles/src/DatafilesToolbar/DatafilesToolbar.tsx

```diff
@@ -58,6 +58,7 @@ export function getToolbarRules(
       isAuthenticated &&
       selectedFiles.length >= 1 &&
       !isReadOnly &&
+      system !== USER_WORK_SYSTEM &&
```

Comment:
I wonder, should we limit trash systems with an approved list, rather than make sure it doesn't match a system? Thinking about it, we probably only want to allow trash in a subset of systems.

Reply:
I think a deny-list approach makes more sense for Designsafe, since we have a limited number of VM/HPC systems and an arbitrary number of project systems targeting cloud.data. I'm approving/merging for now.